pkgs/development/python-modules/mario/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, poetry2nix, fetchFromGitHub, fetchPypi, buildPythonPackage, cffi }:
+
+poetry2nix.mkPoetryApplication rec {
```

Comment:
for `python-modules`, you need to use `buildPythonPackage`.

If you just need to just package the application, that can be done outside of `python-modules`. Please refer to https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonapplication-function